src/common/lib/api.ts

```diff
@@ -96,6 +96,8 @@ export const deleteAccount = (id: string) =>
   ]);
 export const unlock = (password: string) =>
   utils.call<UnlockRes>("unlock", { password });
+export const getBlocklist = (host: string) =>
+  utils.call<{ blocked: boolean }>("getBlocklist", { host });
```

Comment:
what's the best practice here? should this be a named type? (or whatever this is called?) @escapedcat

Reply:
Yeah, better define a type which can be reused and also can be used in unit-tests.  
See how it's done just above with `UnlockRes`